src/transformers/models/rag/retrieval_rag.py

```diff
@@ -594,12 +595,27 @@ def __call__(
             docs, input_strings, prefix, n_docs, return_tensors=return_tensors
         )
 
+        retrived_doc_text=[]
+        retrived_doc_title=[]
+
+        for b_idx in range(len(docs)): 
```

Comment:
Can we maybe also only enable this if a certain flag is set, like `return_tokenized_docs`

Reply:
Added a flag.